### PR TITLE
OGP

### PR DESCRIPTION
**Pull request by iranzo**
Add OpenGraph values to the generated docs

### CODE REVIEW

**Comment by iranzo** [comment]
@qbarrand what we discussed about

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 site_name: Kernel Module Management
 repo_url: https://github.com/kubernetes-sigs/kernel-module-management
 edit_uri: edit/main/docs/mkdocs
+site_description: Kernel Module Management (KMM) is a Kubernetes operator that manages, builds, signs and deploys out-of-tree kernel modules and device
+  plugins on Kubernetes clusters.
```
Comment (ybettan) [inline docs/mkdocs.yml:6]:
```suggestion
site_description: Kernel Module Management (KMM) is a Kubernetes operator that manages, builds, signs and deploys out-of-tree kernel modules and device plugins on Kubernetes clusters.
```

File: docs/overrides/main.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <meta property="og:url" content="{{ page.canonical_url }}" />
+  <!-- <meta property="og:image" content="<url>" /> -->
+  <meta property="og:image:type" content="image/png" />
+  <meta property="og:image:width" content="1200" />
+  <meta property="og:image:height" content="630" />
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:title" content="{{ title }}" />
+  <meta name="twitter:description" content="{{ config.site_description }}" />
+  <!-- <meta name="twitter:image" content="<url>" /> -->
+{% endblock %}
```
Comment (ybettan) [inline docs/overrides/main.html:21]: What is that supposed to do?

Comment (iranzo) [reply docs/overrides/main.html:21]: more details about this: https://ogp.me/

**Pull request merged**